languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,34 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLES_OU_QU_ELLES" name="quelles ou qu'elles" default="temp_off">
+      <rule>
+          <pattern>
+              <token postag="SENT_START"/>
+              <marker>
+                  <token regexp="yes">qu'?</token>
+                  <token>elles</token>
+              </marker>
+              <token postag="V.*" postag_regexp="yes" skip="-1"><exception postag="V.* sub*|V avoir sub.*|V etre sub.*" postag_regexp="yes"></exception></token>
+              <token postag="M fin inte"/>
+          </pattern>
+          <message>"Qu'elles" est une erreur grammaticale dans une phrase interrogative. Souhaitez-vous écrire <suggestion>Quelles</suggestion> ?</message>
```

Comment:
The first sentence of the message is a little broad. Also, I', just not a fan of saying "XY est une erreur grammaticale". Also because in this case, it's not really about grammar.

Saying that a question cannot start with "Qu'elles" is disproven by the exception above. Even if the verb is in the indicative mood, can't there be sentences like "Qu'elles sont belles, je le vois, mais combien elles coûtent ?"?